pkg/auth/dependency/principal/customtoken/claims.go

```diff
@@ -5,24 +5,51 @@ import (
 	"github.com/dgrijalva/jwt-go"
 )
 
-type SSOCustomTokenClaims struct {
-	RawProfile map[string]interface{} `json:"skyprofile"`
-	jwt.StandardClaims
+type SSOCustomTokenClaims jwt.MapClaims
```

Comment:
I think it's better to add comment, instead of just commit message, to describe the reason why `MapClaims` is used instead.